cmd/clusterctl/hack/create-local-repository.py

```diff
@@ -74,6 +74,12 @@
           'type': 'InfrastructureProvider',
           'configFolder': 'test/infrastructure/docker/config/default',
       },
+      'runtime-extension-test': {
```

Comment:
I'll take another look on my next round of review. Just in general feels not ideal to have different names for the test extension in different parts of the codebase

Reply:
if it can help, if you find that there are some places where this is not clear/intuitive, we can document why we are doing exceptions (using tilt-extension)

Reply:
tracking this in https://github.com/kubernetes-sigs/cluster-api/issues/7322